src/pages/guides/using/asset-events-actions.md

```diff
@@ -0,0 +1,514 @@
+# Asset Events Actions
+
+<br/>
+
+## Overview
+
+Asset Events are initiated by actions, which are operations (create, update, move, etc.) made on the [resources](asset-events-glossary#resource) of [assets](asset-events-glossary#asset). You can determine which action has been made on which resource by looking at certain properties in the [Asset Events object](asset-events-object.md).
+
+1. The `event:resources` object contains the Resource Change objects, which describe [resources](asset-events-glossary#resource) that have been impacted by an action. Resources are identified by their associated link relations.
+
+   ![event:resources object in JSON-formatted Asset Event](../img/resources.png)
```

Comment:
we can update this screenshot to use the payload delivered by IO events that have "data" instead of "body".

Reply:
Do you know where I can get such a screenshot?

Reply:
i have cropped "body"